blockly-core/core/ui/block_svg/block_svg_functional.js

```diff
@@ -222,6 +222,20 @@ Blockly.BlockSvgFunctional.prototype.renderDrawRightInlineFunctional_ =
   }
 };
 
+Blockly.BlockSvgFunctional.prototype.updateToColour_ = function(hexColour) {
+  goog.base(this, 'updateToColour_', hexColour);
+
+  if (!this.divider_) {
```

Comment:
Is this a proxy for, "block has been rendered"? If so, could pull out a helper method (in case indicator changes later) e.g.

```
hasBeenRendered = function () {
  return !!this.divider_;
}
```